slackline.c

```diff
@@ -75,7 +75,7 @@ sl_postobyte(struct slackline *sl, size_t pos)
 	size_t byte = 0;
 
 	for (;pos > 0; pos--)
-		byte += grapheme_next_character_break(ptr+byte, sl->blen-byte);
+		byte += grapheme_next_character_break_utf8(ptr+byte, sl->blen-byte);
```

Comment:
please don't do this here.
a libgrapheme update have to be done in a separate change.